<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>303</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBoxNetwork">
     <property name="title">
      <string> Network </string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="labelIP">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Address:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditIP"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelPort">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Port:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBoxPort">
          <property name="maximum">
           <number>65000</number>
          </property>
          <property name="value">
           <number>5660</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBoxActivityTimer">
          <property name="text">
           <string>Activity timer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxRotatorWindowFrame">
       <property name="text">
        <string>Rotator window frame visible (reboot)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxConnectOnStartup">
       <property name="text">
        <string>Connect on startup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxRotatorWindowStartOnStop">
       <property name="text">
        <string>Rotator window start on top</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxRotatorShowMousePointer">
       <property name="text">
        <string>Show mouse pointer (Rotator window)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>groupBoxNetwork</sender>
   <signal>clicked(bool)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>groupboxNetworkClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>pressed()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>btnOKClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>pressed()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>btnCancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>groupBoxNetworkClicked(bool)</slot>
  <slot>groupBoxCOMClicked(bool)</slot>
  <slot>btnOKClicked()</slot>
  <slot>btnCancelClicked()</slot>
 </slots>
</ui>
